builtins/web/fetch/request-response.cpp

```diff
@@ -546,6 +548,17 @@ bool RequestOrResponse::parse_body(JSContext *cx, JS::HandleObject self, JS::Uni
     }
     static_cast<void>(buf.release());
     result.setObject(*array_buffer);
+  } else if constexpr (result_type == RequestOrResponse::BodyReadResult::Blob) {
+    auto bytes = reinterpret_cast<uint8_t *>(buf.get());
+    auto data = std::make_unique<std::vector<uint8_t>>(bytes, bytes + len);
```

Comment:
It's a bit unfortunate that this has to copy the bytes again :/ Maybe open an issue about this and reference it as part of a TODO here?

(Unfortunately the only real option to fix this would I think be to stop using a `vector` in `Blob`, which isn't great. And solidly makes this an "optimization problem for later"—hence the suggestion to open an issue for it instead of fixing it here and now.)

Reply:
Oh, see my comment here:

https://github.com/bytecodealliance/StarlingMonkey/pull/169#discussion_r1862493507

Reply:
Perfect, thank you!

Reply:
As of the latest [commit](https://github.com/bytecodealliance/StarlingMonkey/pull/177/commits/43f2f9120a01080197f84b1984e6fe0e75ad6f2d) the buffer is not copied anymore 🎉